test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class StaticPagesControllerTest < ActionController::TestCase
+
```

Comment:
You can kill this line as per AirBnB's Ruby Style Guide (https://github.com/airbnb/ruby#newline-different-indent)